selfdrive/navd/navd.py

```diff
@@ -269,11 +269,11 @@ def send_instruction(self):
         self.recompute_countdown = 0
       else:
         cloudlog.warning("Destination reached")
-        Params().remove("NavDestination")
 
         # Clear route if driving away from destination
         dist = self.nav_destination.distance_to(self.last_position)
         if dist > REROUTE_DISTANCE:
+          self.params.remove("NavDestination")
```

Comment:
move this to clear route instead?

Reply:
Can't, it's used to temporary hide the route on errors as well. I want to move the param into a capnp field so the ui can use that to know why the route was cleared